src/System.Data.Common/src/System/Data/Common/DBCommandBuilder.cs

```diff
@@ -197,7 +197,7 @@ private void EliminateConflictingNames()
                                 // found duplicate name
                                 // the name unchanged name wins
                                 int iMutatedName = _isMutatedName[j] ? j : i;
-                                Debug.Assert(_isMutatedName[iMutatedName], string.Format(CultureInfo.InvariantCulture, "{0} expected to be a mutated name", _baseParameterNames[iMutatedName]));
+                                Debug.Assert(_isMutatedName[iMutatedName], $"{_baseParameterNames[iMutatedName]} expected to be a mutated name");
```

Comment:
Just to confirm, this should have no affect on the release codegen, correct?

Reply:
> this should have no affect on the release codegen, correct?

Correct; the whole Debug.Assert statement and all of the expressions passed as arguments to it are removed as part of its conditionalness.